apps/extension/client/src/editorWebviewScript.ts

```diff
@@ -168,9 +171,13 @@ const machine = createMachine<WebViewMachineContext, EditorWebviewScriptEvent>(
               false
             );
 
+            /**
+             * We add runningInStatelyExtension to the iframe's src.
+             * This is used by `useVsCode` in the editor to check that the extension is running the web app.
+             * */
```

Comment:
I'm not 100% sure as this would have to be tested but I think that you might already have this information in the Studio and that you don't need to pass it explicitly from here. You could try checking `typeof acquireVsCodeApi !== 'undefined'` there - I believe that it should be a function in VSCode's webviews and it ain't anywhere else :p